casanovo/config.py

```diff
@@ -96,10 +96,22 @@ def __init__(self, config_file: Optional[str] = None):
                 # Remap deprecated config entries.
                 for old, new in _config_deprecated.items():
                     if old in self._user_config:
-                        self._user_config[new] = self._user_config.pop(old)
+                        warning_msg = (
```

Comment:
**nitpick:** I would move this into the `else` to mirror the behavior in the `if`.